mantis-runtime/build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
     api project(':mantis-common')
     api libraries.slf4jApi
     testImplementation libraries.junit4
+    testImplementation libraries.junitJupiter
```

Comment:
isn't junit-jupiter meant for usage with junit5?

Reply:
Yes. It works just fine if we only use the utility assertion methods from Jupiter. But I agree, it's better not to mix JUnit 4 and Jupiter. I have written the `assertThrows` in the testing code instead of pulling junitJupiter. I'll file a follow-up PR to upgrade JUnit 4 to Jupiter in`mantis-runtime` module.